src/range-slider/RangeSlider.tsx

```diff
@@ -0,0 +1,67 @@
+// @ts-ignore
+import React, { InputHTMLAttributes, ReactNode, useState } from "react";
+
+import { VariantProps, cva } from "class-variance-authority";
+import { ChangeEvent, forwardRef } from "react";
+
+const rangeSlider = cva(
+  "appearance-none h-3 rounded-full bg-gray-300 accent-pink-500",
+  {
+    variants: {
+      size: {
+        small: "w-32",
+        medium: "w-64",
+        large: "w-96",
+        unbound: "w-full",
+      },
+    },
+    defaultVariants: {
+      size: "medium",
+    },
+  }
+);
+
+type Props = {
+  onVolumeChange: (volume: number) => void;
+  min?: number;
```

Comment:
These shouldn't be necessary as they should be provided via the InputHTMLAttributes